src/main/java/bio/terra/service/dataset/flight/ingest/IngestCreateScratchParquetFilesStep.java

```diff
@@ -82,8 +82,7 @@ public StepResult undoStep(FlightContext context) {
     AzureStorageAccountResource storageAccountResource =
         workingMap.get(
             CommonMapKeys.DATASET_STORAGE_ACCOUNT_RESOURCE, AzureStorageAccountResource.class);
-    String scratchParquetFile =
-        FolderType.SCRATCH.getPath(workingMap.get(IngestMapKeys.PARQUET_FILE_PATH, String.class));
+    String scratchParquetFile = workingMap.get(IngestMapKeys.PARQUET_FILE_PATH, String.class);
```

Comment:
Variable name nit --

This value is no longer a scratch parquet file, it will be transformed into the scratch parquet file path by the method below.  How about naming it `parquetFilePath` instead like you've done in `IngestCreateParquetFilesStep`?